Corona-Energy-Grid-Frontend/src/app/REST/authentication.service.ts

```diff
@@ -25,7 +27,8 @@ export class AuthenticationService {
         response => {
           var token = response.headers.get("Authorization");
           if (token) {
-            localStorage.setItem(AppConfig.LocalStorageKeys.TOKEN, token);
+            //localStorage.setItem(AppConfig.LocalStorageKeys.TOKEN, token);
```

Comment:
waarom uitgecomment en niet verwijderd?

Reply:
is het nodig om te veranderen van localstorage naar cookies als beiden op dezelfde manier functioneren?